docs/src/forc/commands/forc_init.md

```diff
@@ -15,10 +15,27 @@ forc init [OPTIONS] <PROJECT_NAME>
 
 ## OPTIONS:
 
+`--contract` 
```

Comment:
Why is this here?

Reply:
There's a bug with the documentor I think. If I manually change it back, and then run the documentor again, it fills this in from my other branch...? super weird

Reply:
You might be using `forc` built with your other branch??

Reply:
Right you are 🤦🏻